src/sql/workbench/electron-browser/modelComponents/table.component.ts

```diff
@@ -26,7 +26,7 @@ import { Emitter, Event as vsEvent } from 'vs/base/common/event';
 @Component({
 	selector: 'modelview-table',
 	template: `
-		<div #table style="width: 100%;height:100%" [style.font-size]="fontSize"></div>
+		<div #table style="height:100%;" [style.font-size]="fontSize" [style.width]="width"></div>
```

Comment:
do we have a default width on table component - what if someone doesn't set it?

Reply:
the default width seems to be 100% when it isn't set. For example, the source and target boxes are tables and they're widths are the same as before even though I didn't set the widths.

Reply:
okay that's good.

one last question - why does height say 100% here but still we are able to set the height but not the same with width? is it possible that if we set the width during creating the component we don't have to do this? if that possible I would prefer that.

Reply:
I noticed some places like alertsDialog/jobdialog also setting width for table component

Reply:
The alertsDialog sets the width to a px value, but setting the widths initially doesn't work with percentages since it tries to convert it to a number and we need a percentage since the table should grow or shrink with the size of the window.

Reply:
Ah. got it.
Ok sounds good.. let's get a quick check that this doesn't affect other places for table component and get this in.